internal/gen/_template/schemas.tmpl

```diff
@@ -34,6 +34,13 @@ type {{ $.Name }} struct {
 {{- end }}
 {{/* Define struct */}}
 
+{{/* Define embedded */}}
+{{- else if $.Embedded }}
```

Comment:
In this template we render types depending on ```$.Kind``` value.
```$.Embedded``` it's not Kind. This introduces inconsistency.

Reply:
Merged `Embedded` and `RawBinary` kind into `Embedded` kind for consistency.

Reply:
Create `Stream` type for octet-stream